Pipelines/Templates/tests.yaml

```diff
@@ -14,61 +9,33 @@ parameters:
-
-    # $logFile = Join-Path $(Agent.TempDirectory) "editmode-test-run.log"
-    # New-Item -Path $logFile -ItemType File -Force
-
-    # if ("${{ parameters.UnityVersion }}" -eq "Latest") {
-    #   $proc = Start-UnityEditor -Project "${{ parameters.PathToProject }}" -Latest -RunTests -TestPlatform EditMode -BuildTarget ${{ parameters.BuildTarget }} -BatchMode -Quit -PassThru -LogFile $logFile -EditorTestsResultFile ".\test-editmode-default.xml" -AdditionalArguments "${{ parameters.AdditionalArguments }}"
-    # }
-    # else {
-    #   $proc = Start-UnityEditor -Project "${{ parameters.PathToProject }}" -Version ${{ parameters.UnityVersion }} -RunTests -TestPlatform PlayMode -BuildTarget ${{ parameters.BuildTarget }} -BatchMode -Quit -PassThru -LogFile $logFile -EditorTestsResultFile ".\test-editmode-default.xml" -AdditionalArguments "${{ parameters.AdditionalArguments }}"
-    # }
-
-    # $ljob = Start-Job -ScriptBlock { param($log) Get-Content "$log" -Wait } -ArgumentList $logFile
-
-    # while (-not $proc.HasExited -and $ljob.HasMoreData)
-    # {
-    #     Receive-Job $ljob
-    #     Start-Sleep -Milliseconds 200
-    # }
-    # Stop-Job $ljob
-    # Remove-Job $ljob
-    # Stop-Process $proc
-
-    # Write-Output '====================================================='
-    # Write-Output '              PlayMode Tests Finished                '
-    # Write-Output '====================================================='
-#   displayName: 'Run EditMode tests'
-
-- pwsh: |
-    Write-Host "======================= PlayMode Tests ======================="
-
-    $logFile = Join-Path $(Agent.TempDirectory) "playmode-test-run.log"
-    New-Item -Path $logFile -ItemType File -Force
-
-    $resultFile = Join-Path $(Agent.TempDirectory) "test-results-playmode.xml"
-
-    try {
+  - pwsh: |
+      Write-Host "======================= PlayMode Tests ======================="
+      
+      $logFile = Join-Path $(Build.ArtifactStagingDirectory) "playmode-test-run.log"
+      New-Item -Path $logFile -ItemType File -Force
+      
+      $resultFile = Join-Path $(Build.ArtifactStagingDirectory) "test-results-playmode.xml"
+      New-Item -Path $resultFile -ItemType File -Force
+
+      try {
       if ("${{ parameters.UnityVersion }}" -eq "Latest") {
         $proc = Start-UnityEditor -Project "${{ parameters.PathToProject }}" -Latest -RunTests -TestPlatform PlayMode -BuildTarget ${{ parameters.BuildTarget }} -BatchMode -PassThru -LogFile $logFile -EditorTestsResultFile $resultFile -AdditionalArguments "${{ parameters.AdditionalArguments }}"
       }
       else {
         $proc = Start-UnityEditor -Project "${{ parameters.PathToProject }}" -Version ${{ parameters.UnityVersion }} -RunTests -TestPlatform PlayMode -BuildTarget ${{ parameters.BuildTarget }} -BatchMode -PassThru -LogFile $logFile -EditorTestsResultFile $resultFile -AdditionalArguments "${{ parameters.AdditionalArguments }}"
       }
-
```

Comment:
super nit: can you restore all these removed new lines please 😇 helps with readability

Reply:
Still looks like these are missing in the diff

![image](https://user-images.githubusercontent.com/3580640/197262685-0f8f6802-a2f4-46f3-924f-0d0f39bbf963.png)